cmd/geth/config.go

```diff
@@ -107,12 +109,17 @@ func defaultNodeConfig() node.Config {
 	return cfg
 }
 
+func defaultDashboardConfig() dashboard.Config {
+	return dashboard.DefaultConfig
+}
+
 func makeConfigNode(ctx *cli.Context) (*node.Node, gethConfig) {
 	// Load defaults.
 	cfg := gethConfig{
-		Eth:  eth.DefaultConfig,
-		Shh:  whisper.DefaultConfig,
-		Node: defaultNodeConfig(),
+		Eth:       eth.DefaultConfig,
+		Shh:       whisper.DefaultConfig,
+		Node:      defaultNodeConfig(),
+		Dashboard: defaultDashboardConfig(),
```

Comment:
Just use `dashboard.DefaultConfig` here directly. There's no need to add this extra wrapper.